### ISSUE TITLE
Is the SBOL3 EDAM format simply RDF?

### ISSUE THREAD

**Issue by jakebeal** [SynBioDex/SBOL-specification#488]
I am making SBOL3 Attachments that point to SBOL3 files, but I cannot use the "SBOL" EDAM format http://edamontology.org/format_3725, since that is specifically for the SBOL2 XML format, and my files are not in that format.

As a consequence, I am currently simply using RDF format (http://edamontology.org/format_2376), or sometimes more specifically the N-triples format (http://edamontology.org/format_3256).

Is that sufficient description of SBOL3 as a format, or do we want to ask something more specific, like the UniProtKB RDF format (/http://edamontology.org/format_3771)?

**Comment by cjmyers** [comment]
SBML and CellML do not specify a version.  I suggest that you request EDAM to change the definition to not specify a version, and you use the EDAM term to specify it is SBOL.

**Comment by jakebeal** [comment]
I am not so worried about the version, which could be corrected as you say. The real problem is that the format is a subclass of XML, which is an incorrect description of an N-triples RDF file.

**Comment by cjmyers** [comment]
You can ask for that change as well.

**Comment by jakebeal** [comment]
I don't see how to correctly represent SBOL2 as being a XML-only not-exactly-RDF format and SBOL3 as a genetic RDF format using only a single term. What exactly are you recommending here?

**Comment by cjmyers** [comment]
Just make SBOL a generic RDF format.  SBOL2 was RDF compliant too, even if just a subset of serialization were actively used by the community.

**Comment by jakebeal** [comment]
Is that OK even with all of the re-writing rules that were used in SBOL2?

**Comment by cjmyers** [comment]
Absolutely.  SBOL2 is represented in RDF, and it would validate in RDF validators.

**Comment by jakebeal** [comment]
OK; I will try to take this to EDAM

**Comment by jakebeal** [comment]
I filed an issue: https://github.com/edamontology/edamontology/issues/849
It also looks like we have already had this conversation, came to the same conclusion, and have been waiting for an update from EDAM for just under a year: https://github.com/edamontology/edamontology/issues/752
